docs/proposals/20191017-kubeadm-based-control-plane.md

```diff
@@ -381,23 +408,46 @@ spec:
 
 * Infrastructure templates are expected to be immutable, so infrastructure template contents do not have to hashed in order to detect
   changes.
-   
+
+##### KubeadmControlPlane rollout (using delete-and-create)
```

Comment:
What about having a KubeadmControlPlane rollout which describes once what is common (e.g. the triggered by part), and the fork the description of the execution algorithm according to the upgrade strategy